### PR TITLE
Fix: css for dropdown (ens list table) and chars counter (claim name page)

### PR DESCRIPTION
**Pull request by malaniz**
Fix: css for dropdown (ens list table) and chars counter (claim name page)

### CODE REVIEW

File: src/components/ClaimENSPage/ClaimENSPage.tsx [hunk 1]
```diff
@@ -99,22 +112,16 @@ export default class ClaimENSPage extends React.PureComponent<Props, State> {
...
-                        ? t('claim_ens_page.name_message')
-                        : t('claim_ens_page.repeated_message')
-                    }
+                    message={message}
                     placeholder={t('claim_ens_page.name_placeholder')}
                     action={`${name.length}/${MAX_NAME_SIZE}`}
                     error={isError || (hasNameMinLength(name) && !isValid) || !isAvailable}
                     onChange={this.handleNameChange}
-                    onAction={undefined}
+                    onAction={() => undefined}
```
Comment (nachomazzara) [inline src/components/ClaimENSPage/ClaimENSPage.tsx:124]: Hmmm, I'm not sure about this. @cazala  @nicosantangelo. What is the purpose of `onAction` ? `active = false` ?

File: src/components/ClaimENSPage/ClaimENSPage.css [hunk 2]
```diff
@@ -33,13 +33,18 @@
...
 
-.ClaimENSPage .dcl.field .overlay .ui.button.basic {
+.ClaimENSPage .dcl.section .dcl.field .overlay .ui.button.basic {
   font-size: 17px;
   color: var(--secondary-text) !important;
   cursor: default;
 }
 
+.ClaimENSPage .dcl.section.red .dcl.field .overlay .ui.button.basic {
+  color: red !important;
```
Comment (nachomazzara) [inline src/components/ClaimENSPage/ClaimENSPage.css:43]: Do we have a CSS variable to replace `red` somewhere else? Is `red` a DCL standard color? 

Is the `!important` really necessary?

File: src/components/ClaimENSPage/ClaimENSPage.tsx (hunk 1 above)
Comment (malaniz) [reply src/components/ClaimENSPage/ClaimENSPage.tsx:124]: Fixing this with a class method to not regenerate the lambda on every render.

**Review by nachomazzara — APPROVED** [review]
🚀

**Pull request merged**